kernel/arch/dreamcast/include/arch/rtc.h

```diff
@@ -23,29 +27,78 @@ __BEGIN_DECLS
 
 #include <time.h>
 
-/** \brief  Get the current date/time.
+/** \defgroup rtc Real-Time Clock
+    \brief Real-Time Clock Management 
+
+    Provides an API for fetching and managing the date/time using
+    the Dreamcast's real-time clock. All timestamps are in standard
+    Unix format, with an epoch of January 1, 1970. Due to the fact
+    that there is no time zone data on the RTC, all times are expected
```

Comment:
Not sure if it should be here or in the brief, but there should be at least once place where RTC is glossed. So "Real-Time Clock (RTC) Management"

Reply:
Good call!